edx-olive/edx-platform
# CAM2-309 Manually updates Hebrew translations

Compiled using:

    django-admin.py compilemessages -v0
    ./manage.py lms compilejsi18n --settings=devstack
    ./manage.py cms compilejsi18n --settings=devstack


* Updates "{earned}/{possible} points (graded)" and "...(ungraded)"
* Adds "Answers are displayed within the problem"
* Updates buttons on Support/Help pop-up

**Testing instructions**

Deployed to courses.stage.campus.gov.il.

1. Login and register for the [OpenCraft DemoX course](https://courses.stage.campus.gov.il/courses/course-v1:OpenCraft+DemoX+Demo_Course/about).
1. Change your language to Hebrew using the drop-down at the top of the page.
1. Visit the course links for each of the translation changes below.

* Line 77: Submit an answer to these problems to see the updated translation.
  Ungraded point (singular): [Example Week 1: Getting Started  > Homework - Question Styles (ungraded) > Mathematical Expressions](https://studio.stage.campus.gov.il/container/block-v1:OpenCraft+DemoX+Demo_Course+type@vertical+block@bbfa460a7f7b455ea4e8a6c2310d1f23) 
   ![point (ungraded)](https://user-images.githubusercontent.com/7556571/82528711-8c460000-9b78-11ea-83fa-d9967f62f992.png)

  Ungraded points (plural): [Example Week 1: Getting Started  > Homework - Question Styles (ungraded) > Multiple Choice](https://studio.stage.campus.gov.il/container/block-v1:OpenCraft+DemoX+Demo_Course+type@vertical+block@b3cf567c86a1436397bcda2620c6c498)
   ![points (ungraded)](https://user-images.githubusercontent.com/7556571/82528716-8fd98700-9b78-11ea-8078-5983ef15b636.png)
* Line 79: Submit an answer to these problems to see the updated translation.
   Graded point (singular): [Example Week 1: Getting Started  > Homework - Question Styles (graded) > Mathematical Expressions](https://courses.stage.campus.gov.il/courses/course-v1:OpenCraft+DemoX+Demo_Course/jump_to/block-v1:OpenCraft+DemoX+Demo_Course+type@vertical+block@vertical_0c92347a5c00)
   ![point (graded)](https://user-images.githubusercontent.com/7556571/82531334-57d54280-9b7e-11ea-815d-735c35e58477.png)

  Graded points (plural): [Example Week 1: Getting Started  > Homework - Question Styles (graded) > Multiple Choice](https://courses.stage.campus.gov.il/courses/course-v1:OpenCraft+DemoX+Demo_Course/jump_to/block-v1:OpenCraft+DemoX+Demo_Course+type@vertical+block@54bb9b142c6c4c22afc62bcb628f0e68)
   ![points (graded)](https://user-images.githubusercontent.com/7556571/82560912-802a6480-9bb1-11ea-8a44-f7e0b6c9348c.png)
* Line 80: Click "Show Answer" to see the updated translation.
  [Example Week 1: Getting Started  > Homework - Question Styles (graded) > Mathematical Expressions](https://courses.stage.campus.gov.il/courses/course-v1:OpenCraft+DemoX+Demo_Course/jump_to/block-v1:OpenCraft+DemoX+Demo_Course+type@vertical+block@vertical_0c92347a5c00)
  ![Answers are displayed within the problem](https://user-images.githubusercontent.com/7556571/82531498-ad115400-9b7e-11ea-8f0b-5f853d1716fc.png)
* Lines 85, 86, 87: Click the Support tab at the side of any page to check the updated translations on the buttons.
   ![Support tab buttons](https://user-images.githubusercontent.com/7556571/82531746-12fddb80-9b7f-11ea-9453-0fba9bd86377.png)

**Reviewer**
